### DISCUSSION TITLE
Theme for the fish prompt

### DISCUSSION THREAD

**Discussion by UnclePsymon** [nix-community/stylix#532] [Ideas]
Hi! So, Stylix makes my desktop beautiful, but my builtin fish promt still lookes boring. I don't know how to implement it myself (yet), but maybe a brave soul could do it someday? 🐟 

![image](https://github.com/user-attachments/assets/714dd202-a2f4-46f9-8107-c12f09ea2a0f)

**Comment by trueNAHO** [comment]
Is your prompt based on an upstream plugin?

**Reply by UnclePsymon** [reply]
Turns out it is https://github.com/IlanCosman/tide.

I thought it was builtin. I just realized that the builtin promts actually get themed. But they are not as cool as Tide :)

**Reply by UnclePsymon** [reply]
Well! I can just make Tide use base16 colors in the setup of the plugin! Thank you anyways ❤️

**Reply by trueNAHO** [reply]
Feel free to open a PR theming this fish plugin.
